sycl/include/CL/sycl/properties/buffer_properties.hpp

```diff
@@ -39,6 +39,17 @@ class context_bound
 private:
   sycl::context MCtx;
 };
+
+class mem_channel : public detail::PropertyWithData<
+                        detail::PropWithDataKind::BufferMemChannel> {
+public:
+  mem_channel(cl_uint Channel) : m_Channel(Channel) {}
+  cl_uint get_channel() const { return m_Channel; }
+
+private:
+  cl_uint m_Channel;
```

Comment:
Let's not add any more direct uses of OpenCL:

```suggestion
  uint32_t m_Channel;
```

Reply:
Done